SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -101,7 +101,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bTcpNoDelay(false)
 	, UdpServerDownstreamUpdateIntervalMS(1)
 	, UdpClientDownstreamUpdateIntervalMS(1)
-	, bWorkerFlushAfterOutgoingNetworkOp(false)
+	, bWorkerFlushAfterOutgoingNetworkOp(true)
```

Comment:
Turning this on for the legacy connection changes the performance characteristics - can we just enable it when using spatial view?

Reply:
Sure

Reply:
Actually in that case it's probably better to just make MaybeFlush always flush in spatial view.
I'm happy to try and do something more elegant although the shim removal stuff changes all of this anyway.

Reply:
Nice